packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -187,6 +187,14 @@ export type RendererInspectionConfig = $ReadOnly<{}>;
 
 export type TransitionStatus = FormStatus;
 
+export type ViewTransitionRef = {
```

Comment:
The ecosystem has largely converged on using the "Ref" suffix for the actual ref e.g. `{ current: null }`. So people will probably do `<ViewTransition ref={viewTransitionRef}`>`but then `viewTransitionRef` would be of type `React.Ref<ViewTransitionRef>`. 

Would `ViewTransitionInstance` also work?

Reply:
Yea, in fact, I wanted to call it that but I already named the internal state that. But we can rename that to ViewTransitionState.

Reply:
Renamed.